types/react-native/v0.63/index.d.ts

```diff
@@ -1426,7 +1426,9 @@ export interface TextInputSelectionChangeEventData extends TargetedEvent {
  * @see TextInputProps.onKeyPress
  */
 export interface TextInputKeyPressEventData {
+    eventCount: number;
     key: string;
+    target: number;
```

Comment:
Both of them are [maybe types](https://flow.org/en/docs/types/maybe/) in the [source code](https://github.com/facebook/react-native/blob/36bbd8fa31eceabf0fbf52b8daccc848e9c5830c/Libraries/Components/TextInput/TextInput.js#L118) and should therefore be made optional and changed to accept also `null | undefined`:
```suggestion
    key: string;
    eventCount?: number | null | undefined;
    target?: number | null | undefined;
```

Reply:
In addition: I guess we could add some simple tests?